TodoListManagerTest/src/test/java/StoryTest/Todos/TodosStepDefinition.java

```diff
@@ -0,0 +1,249 @@
+package StoryTest.Todos;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Assertions;
+import org.skyscreamer.jsonassert.JSONAssert;
+
+public class TodosStepDefinition {
+
+    private final String json = "application/json";
+    private final String xml = "application/xml";
+    private Response res;
+    private String jsonString;
+    private String xmlString;
+
+    // ----------- Feature: Get specific todos -------------//
+
+    @Given("I have an existing todo")
+    public void i_have_an_existing_todo() {
+        // 2 todos already exist
+    }
+
+    @When("I request the todo with a specific id {int} in JSON")
+    public void i_request_the_todo_with_a_specific_id_in_json(Integer todoID) {
```

Comment:
Is there a reason why you are using Big Int for id?

Reply:
Changed